client/components/product-card/docs/example.jsx

```diff
@@ -1,35 +1,32 @@
 /**
  * External dependencies
  */
-import React, { Fragment } from 'react';
+import React, { Fragment, useState } from 'react';
 
 /**
  * Internal dependencies
  */
 import ProductCard from '../index';
+import Button from 'components/button';
 
 function ProductCardExample() {
+	const [ selectedProductOption, selectProductOption ] = useState(
+		'jetpack_backup_realtime_monthly'
+	);
+	const [ isPlaceholder, setIsPlaceholder ] = useState( false );
+
 	return (
 		<Fragment>
+			<Button compact onClick={ () => setIsPlaceholder( ! isPlaceholder ) }>
+				Toggle placeholders
+			</Button>
+			<hr />
 			<h3>Product Card - default</h3>
 			<ProductCard
 				title="Jetpack Scan"
-				billingTimeFrame="per year"
-				fullPrice={ 25 }
-				description={
-					<Fragment>
-						Automatic scanning and one-click fixes keep your site one step ahead of security
-						threats. <a href="/plans">More info</a>
-					</Fragment>
-				}
-			/>
-
-			<h3>Product Card - with a discount</h3>
-			<ProductCard
-				title="Jetpack Scan"
-				billingTimeFrame="per year"
-				fullPrice={ 25.99 }
-				discountedPrice={ 16.99 }
+				isPlaceholder={ isPlaceholder }
+				billingTimeFrame={ isPlaceholder ? null : 'per year' }
+				fullPrice={ isPlaceholder ? null : 25 }
```

Comment:
Why do we need to change the value based on if it is placeholder?

Reply:
It's here to simulate a real scenario when there is no price data fetched yet thus the inner elements of the `ProductCard` are empty and have no dimensions (besides of padding/margin).

Reply:
👍